src/components/FootnoteReference.js

```diff
@@ -6,17 +6,13 @@ import FootnoteContext from './footnote-context';
 const FootnoteReference = ({ nodeData: { id, refname } }) => {
   const { footnotes } = useContext(FootnoteContext);
 
-  // Get the ID of the parent of an anonymous footnote reference
-  const getAnonymousFootnote = () => {
-    return Object.keys(footnotes).find(key => {
-      const referent = getNestedValue([key, 'references'], footnotes) || [];
-      return referent.includes(id);
-    });
-  };
-  const ref = refname || getAnonymousFootnote();
+  // the nodeData originates from docutils, and may be incorrect for
+  // anonymous footnoteReferences originating from included files
+
+  let ref = refname || id.replace('id', '');
```

Comment:
[nit] The value of `ref` is never modified, so we can use `const`.
```suggestion
  const ref = refname || id.replace('id', '');
```